arviz/plots/backends/bokeh/elpdplot.py

```diff
@@ -146,12 +147,12 @@ def _plot_atomic_elpd(
     ylabels_shown,
     plot_kwargs,
 ):
-    ax_.cross(
-        np.asarray(xdata),
-        np.asarray(ydata),
-        line_color=plot_kwargs.get("color", "black"),
-        size=plot_kwargs.get("s"),
-    )
+    marker = _validate_marker("bokeh")(plot_kwargs.get("marker"))
```

Comment:
`plot_kwargs.get("marker", rcParams["plot.bokeh.marker"])` otherwise the value stored in rcParams is never used.